weave/trace_server/reference/server.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import APIRouter, FastAPI
+
+from weave.trace_server.reference.generate import (
+    ServiceDependency,
+    generate_routes,
+    noop_trace_server_factory,
+)
+
+server_dependency = ServiceDependency(service_factory=noop_trace_server_factory)
+trace_service_router = generate_routes(APIRouter(), server_dependency)
+app = FastAPI()
+app.include_router(trace_service_router)
+
+if __name__ == "__main__":
+    import uvicorn
+
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Comment:
not blocking, just curious: do you actually need to start the server here?

Reply:
Nope, but it doesn't hurt and is added for convenience in case you want to run the `server.py` file directly